Ogra/Encodable.swift

```diff
@@ -131,12 +131,8 @@ extension JSON {
 		case .Array(let array):  return array.map { $0.JSONObject() }
 		case .Object(let object):
 			var dict: [Swift.String : AnyObject] = [:]
-			for key in object.keys {
-				if let value = object[key] {
-					dict[key] = value.JSONObject()
-				} else {
-					dict[key] = NSNull()
-				}
+			for (key, value) in object {
+                dict[key] = value.JSONObject()
```

Comment:
The original code deliberately (I really should add some comments to that effect) outputs a `null` entry if there is a key in the incoming dictionary. The rationale being that if the caller has gone to the trouble of adding a null value to the dictionary, then we should go to the trouble of encoding it. This is behaviour that I don't want to lose.